elfi/methods/bo/acquisition.py

```diff
@@ -109,30 +136,54 @@ class LCBSC(AcquisitionBase):
     would be t**(2d + 2).
     """
     
-    def __init__(self, *args, delta=.1, **kwargs):
+    def __init__(self, *args, delta=0.1, **kwargs):
         super(LCBSC, self).__init__(*args, **kwargs)
         if delta <= 0 or delta >= 1:
             raise ValueError('Parameter delta must be in the interval (0,1)')
         self.delta = delta
 
-    def beta(self, t):
+    def _beta(self, t):
         # Start from 0
         t += 1
         d = self.model.input_dim
         return 2*np.log(t**(2*d + 2) * np.pi**2 / (3*self.delta))
 
     def evaluate(self, x, t=None):
-        """ Lower confidence bound selection criterion = mean - sqrt(\beta_t) * std """
+        """Lower confidence bound selection criterion: 
+        
+        mean - sqrt(\beta_t) * std
+        
+        Parameters
+        ----------
+        x : numpy.array
+        t : int
+            Current iteration (starting from 0).
+        """
         if not isinstance(t, int):
             raise ValueError("Parameter 't' should be an integer.")
 
-        mean, var = self.model.predict(x, noiseless=True)
-        return mean - np.sqrt(self.beta(t) * var)
+        mean, var = self.model.predict(x)
+        return mean - np.sqrt(self._beta(t) * var)
+
+    def evaluate_grad(self, x, t=None):
```

Comment:
Ok, I'll change all to grad, since even [Wikipedia](https://en.wikipedia.org/wiki/Gradient) says: "The notation grad f is also commonly used for the gradient."